db/migrate/20180624155644_add_show_dinkurs_events_column_to_company.rb

```diff
@@ -0,0 +1,5 @@
+class AddShowDinkursEventsColumnToCompany < ActiveRecord::Migration[5.1]
+  def change
+    add_column :companies, :show_dinkurs_events, :boolean
```

Comment:
This creates a field which defaults to `nil`, which is OK for companies that do not use this feature yet.  However, we have 5 companies in production who are already using this feature.

Here is the dinkurs-fetch log from last night:
```
[SHF_TASK] [Load Dinkurs Events] [info] Started at 2018-06-27 01:30:07 UTC
[SHF_TASK] [Load Dinkurs Events] [info] Company 24: 80 events.
[SHF_TASK] [Load Dinkurs Events] [info] Company 61: 3 events.
[SHF_TASK] [Load Dinkurs Events] [info] Company 89: 36 events.
[SHF_TASK] [Load Dinkurs Events] [info] Company 108: 2 events.
[SHF_TASK] [Load Dinkurs Events] [info] Company 116: 8 events.
[SHF_TASK] [Load Dinkurs Events] [info] Company 165: 0 events.
[SHF_TASK] [Load Dinkurs Events] [info] Company 214: 0 events.
[SHF_TASK] [Load Dinkurs Events] [info] Finished at 2018-06-27 01:30:12 UTC.
[SHF_TASK] [Load Dinkurs Events] [info] Duration: 4.94 seconds.
```
So, this migration has to set this value to `true` for any company for which `dinkurs_company_id.blank?` is _not_ true (this will prevent setting the attribute to `true` for the two companies that have a blank string for that attribute).